lesson_07/conditionals/src/lesson7.ts

```diff
@@ -7,6 +7,9 @@ import { computeLexicographicDistance } from "./util.js";
  * @return True if the age corresponds to a voting age and false otherwise.
  */
 export function canVote(age: number): boolean {
+  if (age >= 18){
```

Comment:
Fun fact: you could also just do `return age >= 18;` and make this a one-liner.

Reply:
This makes it much more clean. Thank you!!